syntax/fluent.ebnf

```diff
@@ -1,41 +1,53 @@
-body                 ::= (entry NL)* entry? EOF
+body                 ::= (_* NL)* (entry NL)* entry? EOF
 entry                ::= comment
                        | section
                        | message
 
-comment              ::= ('//' [^#xA#xD]* NL)+
-section              ::= '[[' __ word (__ word)* __ ']]'
+comment              ::= ('//' (char - NL)* )+
+section              ::= '[[' _ word (_ word)* _ ']]'
 
 char                 ::= [https://www.w3.org/TR/REC-xml/#NT-Char]
-__                   ::= [#x20#x9]*   /* space, tab */
-NL                   ::= [#xA#xD]+    /* line feed, carriage return */
+
+/* special chars: { } [ \ ] */
+special              ::= [#x5b#x5c#x5d#x7b#x7d]
+
+/* line feed, carriage return; space, tab */
+line-break           ::= [#xA#xD]+
+inline-space         ::= [#x20#x9]+
+
+/* if you break lines you need to indent afterwards */
+break-indent         ::= (line-break* inline-space)+
+
+_                    ::= inline-space
+NL                   ::= line-break
+__                   ::= break-indent
```

Comment:
Are you asking about why `inline-space` and `break-indent` have aliases? If so, the answer is simply: readability of other productions.